src/app/containers/Headings/__snapshots__/index.test.jsx.snap

```diff
@@ -501,6 +507,7 @@ exports[`Headings with rich text with different attributes should render headlin
 >
   <h1
     className="c1"
+    id={null}
```

Comment:
React handles these on the front end to remove them so we don't need to worry about `id="null"` appearing.